R/robust-lmrob-tidiers.R

```diff
@@ -22,12 +22,35 @@
 #' @export
 #' @family robust tidiers
 #' @seealso [robust::lmRob()]
-#' @include stats-lm-tidiers.R
-tidy.lmRob <- function (x, ...) {
-  dots <- enquos(...)
-  dots$conf.int <- FALSE
-  
-  rlang::exec(tidy.lm, x, !!!dots)
+tidy.lmRob <- function (x, quick = FALSE,...){
+  if (quick) {
```

Comment:
I recently decided that I no longer want to support `quick` as an option and am going to remove it throughout `broom`. Would you remove the `quick` logic here?